src/builder.rs

```diff
@@ -403,10 +403,12 @@ impl Builder {
             //     || SpendInfo::dummy(asset, &mut rng),
             //     |s| s.create_split_spend(),
             // );
-            let dummy_spend = SpendInfo::dummy(asset, &mut rng);
 
             // Extend the spends and recipients with dummy values.
-            spends.extend(iter::repeat_with(|| dummy_spend.clone()).take(num_actions - num_spends));
```

Comment:
Do not remove those lines but add them to the comment starting in line 400.
Will will need to uncomment everything at later time to properly incorporate the split_note functionality.